x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -20,8 +20,10 @@ import type { MaybeImmutable } from '../../types';
  */
 export const calculateEndpointAuthz = (
   licenseService: LicenseService,
-  fleetAuthz: FleetAuthz | undefined, // TODO: Remove `undefined` type when `fleetAuthz` is needed and used.
-  userRoles: MaybeImmutable<string[]>
+  fleetAuthz: FleetAuthz,
+  userRoles: MaybeImmutable<string[]>,
+  // to be used in follow-up PRs
+  isEndpointRbacEnabled: boolean = false
```

Comment:
Is there a specific reason to have this as a parameter and don't get it inside?
```typescript
const isEndpointRbacEnabled = ExperimentalFeaturesService.get().endpointRbacEnabled;
```

Reply:
this is a common service but `ExperimentalFeaturesService` is a public service and gets `init` in the public plugin.